src/NuGet.Core/NuGet.PackageManagement/NuGet.PackageManagement.csproj

```diff
@@ -1,9 +1,19 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
+  <!--
+    Import the SDK props manually _after_ we've imported the build customizations that
+    set things like IntermediateOutputPath, OutputPath, etc. This is required due to
+    https://github.com/Microsoft/msbuild/issues/2245. Without applying this fix,
+    a reference to netstandard.dll is never brought in, and the build fails.
+  -->
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
+    <NetStandardPackageVersion>2.0.1</NetStandardPackageVersion>
+    <NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard2.0' ">$(NetStandardPackageVersion)</NetStandardImplicitPackageVersion>
```

Comment:
I don't think this is necessary.  If you use a current version of the .NET Core SDK you will get this automatically.

Reply:
Entirely possible. I had a lot of issues getting things to build correctly, though much of that may have been due to the bug that the manual SDK props import is working around.